schemas/iso19139/src/main/plugin/iso19139/formatter/xsl-view/view.xsl

```diff
@@ -983,7 +984,7 @@
 
   <!-- Link to other metadata records -->
   <xsl:template mode="render-field"
-                match="*[@uuidref]"
+                match="srv:operatesOn[@uuidref]"
```

Comment:
So probably make sense to add also gmd:featureCatalogueCitation, gmd:source and  gmd:aggregateDataSetIdentifier/*/gmd:code. Thanks